### PR TITLE
BlogHome and BlogEntry RIDDL definitions

### LINKED ISSUE #33: Blog Home Page RIDDL definition

### LINKED ISSUE #34: Blog Entry Page RIDDL definition

### PR DESCRIPTION
**Pull request by JamesYopp**
closes #33 
closes #34 
RIDDL definitions for Blog Home and Blog Entry application pages

### CODE REVIEW

File: design/src/main/riddl/application/pages/blogEntry.riddl [hunk 1]
```diff
@@ -1 +1,12 @@
-page BlogEntryPage { ??? }
+include "fragments/header"
+
+type BlogHomeNav is String
+
+command GotoBlogHome is {???}
```
Comment (reid-spencer) [inline design/src/main/riddl/application/pages/blogEntry.riddl:5]: ??? needs to be completed

File: design/src/main/riddl/application/pages/blogHome.riddl [hunk 2]
```diff
@@ -1 +1,35 @@
...
+  blogList: BlogSummary*
+}
+
+record BlogEntry is {
+  title: String,
+  heroImage: URL,
+  intro: String,
+  content: String
+}
+
```
Comment (reid-spencer) [inline design/src/main/riddl/application/pages/blogHome.riddl:25]: The strings above can't all be infinite lengthen. I suggest you add minimum and maximum length specifiers.

File: design/src/main/riddl/application/pages/fragments/header.riddl [hunk 3]
```diff
@@ -1 +1,19 @@
...
+type OrderDashboardLink is String
+type AboutUsLink is String
+type ResourcesLink is String
+
+command GotoPricingPage is {???}
+command GotoCustomerDashboard is {???}
+command GotoOrderDashboard is {???}
+command GotoAboutUsPage is {???}
+command GotoResources is {???}
+
```
Comment (reid-spencer) [inline design/src/main/riddl/application/pages/fragments/header.riddl:12]: These commands need field definitions

**Review by reid-spencer — DISMISSED** [review]
LGTM. I left a few individual comments about incomplete command definitions and unbounded strings but that shouldn't prevent merging this.

**Review by antagonist112358 — APPROVED** [review]
Because

**Review by AlexWeinstein92 — APPROVED** [review]

**Pull request merged**